Makefile

```diff
@@ -169,7 +169,7 @@ clean-raxml:
 # module so that Make can do its job.
 install-perlModules:
 	@echo "Installing Perl modules using cpanminus"
-	for package in Config::Simple File::Slurp Math::Round Number::Range Statistics::Distributions Statistics::Basic Graph::Centrality::Pagerank String::Escape Statistics::LineFit; do \
+	for package in Bio::Perl Bio::FeatureIO Config::Simple File::Slurp Math::Round Number::Range Statistics::Distributions Statistics::Basic Graph::Centrality::Pagerank String::Escape Statistics::LineFit; do \
```

Comment:
Installation for Bio::Perl and Bio::FeatureIO is cumbersome and so I'll leave this part out.  It is up to the user to install these.